src/main/scala/rx/lang/scala/Observable.scala

```diff
@@ -5042,6 +5043,24 @@ object Observable {
   }
 
   /**
+   * Converts a `Try` into an `Observable`.
+   *
```

Comment:
There's some info on the marble diagrams [here](https://github.com/ReactiveX/reactivex.github.io#marble-diagrams), but I don't think we need one here, converting a `Try` into an `Observable` is quite a straightforward operation I guess :wink: